src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx

```diff
@@ -58,12 +59,18 @@ export default function PaginatedIntegrationsList({
 		if (isFirstRender.current) {
 			isFirstRender.current = false
 		} else {
-			// Scroll to the top of the page
-			window.scrollTo(0, 0)
 			// Try to find the first result link, and focus it
 			const targetElement = containerRef.current?.querySelector('a')
 			if (targetElement) {
-				targetElement.focus()
+				targetElement.focus({ forceVisible: true })
```

Comment:
Since you have access to the `targetElement` can you use `targetElement.scrollIntoView()`?

Reply:
I think `.focus()` does this automatically (in most browsers, at least). As Ashlee mentioned, I think the issue in the screengrab I shared earlier was that while we're already scrolling the element into view, a good portion of it is hidden behind the stick top nav bar.

I've updated to grab the (brittle, imperfect, but still useful) `--navigation-header-height` variable, and added that to a `scrollBy` offset (as Ashlee suggested! 💯 ). I guess we could also do `targetElement.scrollIntoView()` before that `scrollBy` as a kind of insurance, but I don't think it's necessary.

Reply:
ah interesting, TIL scrollBy